apps/search/src/assets/i18n/en.json

```diff
@@ -1,8 +1,10 @@
 {
 	"My Geonetwork Catalog": "My Geonetwork Catalog",
 	"Find your metadata": "Find your metadata",
-	"loadResults": "Load results",
 	"Sort by": "Sort by",
+	"readMore": "",
```

Comment:
@fgravin should we be mode specific on translation keys ? In master we have sometimes key that we are using in different part of the application and at some point, you want different translations because used in different context.
eg. `readMore` here is to have more info about the record and can be something slightly different somewhere else in the app.

Also translations are specific to a component or can be shared ?

Should we have a more fine grained conventions eg. `results.readMore`, `results.readMore.tooltip` for tooltips, `results.readMore.help` for helper text ... `sortRecordBy`, `sortRecordBy.resourceTitle`, ...

Some times we have camelCase key, sometime words eg. "last changed", sometimes capital letters, sometimes not...
Using english as key may not be a good idea, if there is a typo, then we have the key with a typo that may be painful to fix ?
Also related to that in master we sometimes have `:` in templates, sometimes in translations - what is best?

I'm raising this here, but maybe a dedicated discussion should be added to an issue?